experimental/cloudstrapper/playbooks/build-configure.yaml

```diff
@@ -0,0 +1,21 @@
+---
+
+- hosts: "{{ buildnode }}"
+  roles:
+    - { role: build-platform, tags: platform }
+  vars:
+    awsTargetRegion: "{{ awsOrc8rRegion }}"
```

Comment:
one minor comment. I see use of both camel casing and underscores. If we can use one convention that will be great. Personally i feel using underscores for variable naming suits ansible and python.

Reply:
Agree with the overall idea of using a consistent naming formation. However, in this case, I have used underscores where the underlying tool (in this case ansible), backend application (a shell script that use environment variables) or general platform related conventions (linux environment variables).